compose/live-view.yml

```
@@ -12,7 +12,7 @@ services:
       liveView: "true"
       fullscreenView: "true"
       showVideoDebugStats: "true"
-      DISPLAY: "unix:0"
+      DISPLAY: "unix${DISPLAY}"
```

Shouldn't it be 

```suggestion
      DISPLAY: "${DISPLAY}"
```

or is `unix` a specific host created elsewhere?

---

Not sure off the top of my head, we should add a story to go through this code to clean it up.

---

OK. My understanding is X uses `DISPLAY` in the form `[host]:configuration[.screen]`, where `host` defaults to the local machine and `screen` defaults to `0`. That is, if the goal is to get the same `DISPLAY` from the host environment into the container, it probably should be the variable unmodified, or perhaps the `host` should be resolvable to the host machine. I'm not familiar enough with this particular service to know where this comes up.